### ISSUE TITLE
Automatically add packages based on import/require statements

### ISSUE THREAD

**Issue by jfsiii** [christianalfoni/webpack-bin#16]
It'd be nice if you could just type `import React from 'react'` and have webpack-bin do the right thing for you.

**Comment by christianalfoni** [comment]
Indeed, though not sure how it works. I see that esnextb.in does this. But they only support import statements. With this tool you also have require statements. You can even lazy load packages with `require.ensure`. So yeah, not quite sure how to solve it.

**Comment by bfitch** [comment]
Could you do it for just `import` statements?

**Comment by developit** [comment]
I believe when I looked into this, esnextb.in is actually looking at the transpiled code for require statements using [detective](https://www.npmjs.com/package/detective). It also looks like someone has set up a wrapper that handles both cases: [detect-import-require](https://www.npmjs.com/package/detect-import-require).

**Comment by christianalfoni** [comment]
That is awesome! Thanks for the tip! Will add this to release 11. I suspect this will run just fine in the browser, meaning that it gives some flexibility in terms of user experience :-)
